.eslintrc.js

```diff
@@ -19,6 +17,7 @@ module.exports = {
   rules: {
     semi: ['error', 'never'],
     'no-trailing-spaces': ['error', { skipBlankLines: true }],
-    'comma-dangle': ['error', 'never']
+    'comma-dangle': ['error', 'never'],
+    'linebreak-style': ['error', 'windows']
```

Comment:
Please set unix as it's more consistent with other OSes and supported by windows anyway afaik

Reply:
```suggestion
    'linebreak-style': ['error', 'unix']
```

Reply:
Unfortunately, your suggestion doesn't work on windows:
![eslint](https://user-images.githubusercontent.com/113309294/221996954-c5e6129a-534e-4035-acbb-e574cbc96e17.png)

It wouldn't be such a big deal if I had to change it once but this comes back everytime I switch branches which is so annoying. I changed all VSC setting related to the default line break but they don't apply, don't know why.

Reply:
Did you set in the bottom right corner the end of line style to CRLF?